src/components/Collection/TimelineSidebar.tsx

```diff
@@ -17,42 +16,36 @@ const TimelineItem = ({ series }: { series: SeriesType }) => {
     : series.AniDB?.Type;
 
   return (
-    <Link to={`/webui/collection/series/${series.IDs.ID}`}>
-      <div className="flex gap-x-3" key={series.IDs.ID}>
+    <div className="flex gap-x-3" key={series.IDs.ID}>
+      <Link to={`/webui/collection/series/${series.IDs.ID}`}>
```

Comment:
This would cause the title to not have a link..? 🤔

Reply:
Yep, the same as the current collection list view so that the whitespace isn't a clickable link and for more easily selectable text. Adding a separate link to the title sounds good though so I did that and applied it to the list view too. For consistency's sake, I added similar behaviour to the poster view so that hovering over whitespace won't activate the poster zoom.